cfgov/jinja2/v1/_includes/organisms/third-width-link-blob-group.html

```diff
@@ -20,4 +20,4 @@
 
 {% from 'organisms/info-unit-group.html' import info_unit_group with context %}
 
-{{ info_unit_group( blocks=value.link_blobs, columns=3, heading=value.heading) }}
+{{ info_unit_group(heading=value.heading, blocks=value.link_blobs, columns=3) }}
```

Comment:
You could add `columns` into the code comment while you're in here if you wanted.